uint8 state

uint8 SHOOTER_MOVING=0
uint8 SHOOTER_POS0=1
uint8 SHOOTER_POS1_UP=2
uint8 SHOOTER_POS1_DOWN=3
uint8 SHOOTER_POS2_UP=4
uint8 SHOOTER_POS2_DOWN=5
uint8 SHOOTER_POS3_UP=6
uint8 SHOOTER_POS3_DOWN=7
